From a9a1e65d992e5eb2563f64ef8596017e9b817f5d Mon Sep 17 00:00:00 2001
From: Sherman Siu <s8siu@uwaterloo.ca>
Date: Sat, 20 Sep 2025 03:53:35 -0400
Subject: [PATCH] Remove unneeded dependencies

- These dependencies are moved to the build/host dependencies for the Conda package. They are not needed at runtime.
---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index 3949ea8..c054d34 100644
--- a/setup.py
+++ b/setup.py
@@ -56,15 +56,12 @@ cmdclass: Mapping[str, type[setuptools.Command]] = {}
 install_requires = [
     "numpy",
     "torch",
-    "ninja",
     "requests",
-    "pynvml",
     "einops",
     "click",
     "tqdm",
     "tabulate",
     "packaging>=24.2",
-    "nvidia-cudnn-frontend>=1.13.0",
 ]
 generate_build_meta({})
 
